app/helpers/passport.js

```diff
@@ -19,36 +22,87 @@ module.exports = function(passport) {
     passport.use(new GitHubStrategy({
         clientID: config.authentication.github.clientId,
         clientSecret: config.authentication.github.clientSecret,
-        callbackURL: '/login/callback'
+        scope: 'repo, user'
     },
-    function(accessToken, refreshToken, profile, done) {
-        process.nextTick(function () {
-            User.findOne({ 'id':  profile.id }, function(err, user) {
-                if (err) {
-                    return done(err);
-                }
+    function(accessToken, refreshToken, githubProfile, done) {
+        User.findOne({ 'githubId': githubProfile.id }, function(err, user) {
+            if (err) {
+                return handleError(err);
+            }
+
+            if (user) {
+                return done(null, user);
+            } else {
+                var newAccount = new Account({
+                    name: 'hashtag-include'
```

Comment:
Nope, not missing anything. A lot of account logic still needs to be fleshed out, so it's just hardcoded right now, I should have just set it to null instead. I'll update it